packages/jazz-tools/src/coValues/coStream.ts

```diff
@@ -50,44 +50,97 @@ export type SingleCoStreamEntry<Item> = {
   tx: CojsonInternalTypes.TransactionID;
 };
 
-/** @category CoValues */
+/**
+ * CoStreams are collaborative logs of data.
+ *
+ * @categoryDescription Content
+ * They are similar to `CoList`s, but with a few key differences:
+ * - They are append-only
+ * - They have a per-session view
+ *
+ * ```ts
+ * favDog.push("Poodle");
+ * favDog.push("Schnowzer");
+ * ```
+ *
+ * @category CoValues
+ */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export class CoStream<Item = any> extends CoValueBase implements CoValue {
+  /**
+   * Declare a `CoStream` by subclassing `CoStream.Of(...)` and passing the item schema using a `co` primitive or a `co.ref`.
+   *
+   * @example
+   * ```ts
+   * class ColorStream extends CoStream.Of(co.string) {}
+   * class AnimalStream extends CoStream.Of(co.ref(Animal)) {}
+   * ```
+   *
+   * @category Declaration
+   */
   static Of<Item>(item: IfCo<Item, Item>): typeof CoStream<Item> {
     return class CoStreamOf extends CoStream<Item> {
       [co.items] = item;
     };
   }
 
+  /**
+   * The ID of this `CoStream`
+   * @category Content */
   declare id: ID<this>;
+  /** @category Type Helpers */
   declare _type: "CoStream";
   static {
     this.prototype._type = "CoStream";
   }
+  /** @category Internals */
   declare _raw: RawCoStream;
 
   /** @internal This is only a marker type and doesn't exist at runtime */
   [ItemsSym]!: Item;
+  /** @internal */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   static _schema: any;
+  /** @internal */
   get _schema(): {
     [ItemsSym]: SchemaFor<Item>;
   } {
     return (this.constructor as typeof CoStream)._schema;
   }
 
+  /**
+   * The per-account view of this `CoStream`
```

Comment:
Adding an example here would be good to make clear how you can
- directly index the CoFeed by account ID
- use something like Object.entries on it

Reply:
Added some examples